### PR TITLE
[PATCH v8] github_ci: replace centos 8 with rocky linux 8 and remove ubuntu 16.04 images

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: .github/workflows/ci-pipeline-arm64.yml [hunk 1]
```diff
@@ -39,12 +39,15 @@ jobs:
       fail-fast: false
       matrix:
         cc: [gcc, clang]
-        os: ['ubuntu_16.04', 'ubuntu_18.04']
+        os: ['ubuntu_16.04', 'ubuntu_18.04', rocky_linux_8]
```
Comment (TuomasTaipale) [inline .github/workflows/ci-pipeline-arm64.yml]: `rocky_linux_8` could be in quotes for symmetry.

File: .github/workflows/ci-pipeline-arm64.yml [hunk 2]
```diff
@@ -39,12 +39,15 @@ jobs:
...
     steps:
       - uses: AutoModality/action-clean@v1.1.0
       - uses: actions/checkout@v2
       - run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g -e CC="${{matrix.cc}}"
                -e CONF="${CONF}" $CONTAINER_NAMESPACE/odp-ci-${{matrix.os}}-${ARCH}-native /odp/scripts/ci/build_${ARCH}.sh
+      - name: Failure log
```
Comment (TuomasTaipale) [inline .github/workflows/ci-pipeline-arm64.yml:48]: Not all jobs have this "Failure log" step, is this on purpose?

Comment (MatiasElo) [reply .github/workflows/ci-pipeline-arm64.yml:48]: Yep, I'm trying to figure out why configure is failing on arm64.

Comment (TuomasTaipale) [reply .github/workflows/ci-pipeline-arm64.yml:48]: OK. Point here was that shouldn't all jobs have the "Failure Log" step, why not have it?

File: .github/workflows/ci-pipeline-arm64.yml (hunk 1 above)
Comment (MatiasElo) [reply .github/workflows/ci-pipeline-arm64.yml]: Fixed.

File: .github/workflows/ci-pipeline-arm64.yml (hunk 2 above)
Comment (MatiasElo) [reply .github/workflows/ci-pipeline-arm64.yml:51]: Added to all build tests.

**Review by TuomasTaipale — APPROVED** [review]

**Review by psavol — APPROVED** [review]

**Pull request merged**